ProjectLighthouse.Servers.Website/Pages/Moderation/NewCasePage.cshtml.cs

```diff
@@ -28,8 +31,15 @@ public IActionResult OnGet([FromQuery] CaseType? type, [FromQuery] int? affected
         if (affectedId == null) return this.BadRequest();
 
         this.Type = type.Value;
+
         this.AffectedId = affectedId.Value;
+        this.AffectedUser = await this.Database.Users.FirstOrDefaultAsync(u => u.UserId == this.AffectedId);
```

Comment:
This line and the associated null check should probably be before you pull their case history